setup.cfg

```diff
@@ -34,6 +34,7 @@ markers =
     cloud:Run the cloud tests for example
 filterwarnings =
     error::FutureWarning
+    error:Detected call of `lr_scheduler.step\(\)` before `optimizer.step\(\)`:UserWarning
```

Comment:
I added this line so that our CI fails if this warning appears. This way it tests that your patch works as expected.

Reply:
Thanks, but this also makes IPU tests fail, this PR is focused on GPU. Not sure where to fix the issue on IPUs

Reply:
@milesial IMHO, we could just skip the IPU test failures as done in my old PR #11755 for now.